tests/test_client/tests.py

```diff
@@ -73,6 +74,13 @@ def test_get_view(self):
         self.assertEqual(response.context['var'], '\xf2')
         self.assertEqual(response.templates[0].name, 'GET Template')
 
+    def test_pickle_resp(self):
+        # neglect non-pickleable attributes in response
```

Comment:
`test_pickle_resp` => `test_pickling_response` or something?

Promote comment to docstring, `A response be serialised and deserialised using pickle` or something? 

Possibly the whole test method should be somewhere else, but I'll leave that for @felixxm to say, because I sure don't know where :)

Reply:
Please also introduce a new view in the URLConf, which is a Class Based View, and mount it to eg: `/cbv_view/` and use that as a pickle test too. It should fail before the patch and succeed after the patch. 

The ancient lineage of the ticket has its roots in the fact that CBVs are stored into the `ResolverMatch` which caused errors. Nowadays thanks to ticket 23895 being resolved by #14664, both function and class views should error, but it would be nice to put it to bed by demonstrating both types are consistently pickled.